qcore/binary_version.py

```diff
@@ -27,9 +27,12 @@ def get_lf_bin(version, bin_loc=default_bin_loc):
 
 
 def get_genslip_bin(version, bin_loc=default_bin_loc):
-    return path.join(bin_loc,  GENSLIP_NAME_STR.format(version))
+    return path.join(bin_loc, GENSLIP_NAME_STR.format(version))
 
 
 def get_unversioned_bin(bin_name, bin_loc=default_bin_loc):
     return path.join(bin_loc, bin_name)
 
+
+def get_opensees_bin():
+    return qconfig["OpenSees"]
```

Comment:
As OpenSees is a tool we build for our own use, this should be joined to bin_loc, and the machine config changed to not have the full path.
If a full path is required (Say for an external build), then the machine config can have the full path, and join will discard the bin_loc.

Reply:
as opensees is a seperate external source code, i dont think putting it under the emod3d directory makes too much sense?
Its just like gmt, where we have a seperate opt directroy for it, as well as its dependencies.

Reply:
Hmm, ok, I think this will be fine. I have some thoughts on how non emod3d repo binaries should be handled, but that's beyond the scope of this PR